### PR TITLE
simple uuid v4 printing

### PR DESCRIPTION
**Pull request by xieyuheng**
I stabbed a pr there, just to avoid "Hello, World!" for now.
I wish you can merge it.
(I only know how to use v4)

### CODE REVIEW

File: src/bin/uuid.rs [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#![feature (uniform_paths)]
+
+use clap as cmd;
+use uuid::{
+    Uuid,
+};
+
+fn print_uuid (v: &str) {
+    match v {
+        // "1" => {
```
Comment (KodrAus) [inline src/bin/uuid.rs:10]: Let's just remove this commented out code and stick with random v4 ids for now.

File: src/bin/uuid.rs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        eprintln! ("- fail to parse -n : {}", n);
+        eprintln! ("  expecting an usize number");
+        std::process::exit (1);
+    }
+}
+
+fn main () {
+    let matches = cmd::App::new ("uuid")
+        .version (cmd::crate_version! ())
+        .arg (cmd::Arg::with_name ("-v")
```
Comment (KodrAus) [inline src/bin/uuid.rs]: We probably don't need an argument for `-v` since there's only a single value it can take.

**Review by KodrAus — COMMENTED** [review]
Thanks @xieyuheng!

I don't think we need to worry about supporting other `Uuid` versions besides `v4` at this stage because the others need some input from the caller.

**Comment by Dylan-DPC-zz** [comment]
@xieyuheng why did you close this? it is a good first step and you just need to make the changes.

**Pull request closed without merging**